stripe/src/main/java/com/stripe/android/model/CardBrand.kt

```diff
@@ -74,7 +79,13 @@ enum class CardBrand(
         "jcb",
         "JCB",
         R.drawable.stripe_ic_jcb,
-        pattern = Pattern.compile("^(35)[0-9]*$")
+        pattern = Pattern.compile("^(35)[0-9]*$"),
```

Comment:
As a follow up to my comment above, this should then be `^(352[89]|35[3-8])[0-9]*` instead, or any card number starting with 35 at a length without a partial pattern is going to show up as JCB.

Reply:
good catch, somehow I missed this